src/libs/OptionsListUtils.js

```diff
@@ -541,7 +541,7 @@ function getOptions(reports, personalDetails, draftComments, activeReportID, {
  * @param {Object} reports
  * @param {Object} personalDetails
  * @param {String} searchValue
- * @param {Array<String>} betas
+ * @param {Array<String>=} betas
```

Comment:
Whats the `=` doing here? Is this intentional?

Reply:
Interesting.. I don't get that error and use vscode - perhaps you have some global rules set up.

Reply:
Forgot this one

Reply:
I think was this [extension](https://marketplace.visualstudio.com/items?itemName=dbaeumer.vscode-eslint) which showed me that warning in vscode 👍🏽

Reply:
Let's remove it?

Reply:
Ok sure, I will revert it